components/eventing-controller/README.md

```diff
@@ -135,69 +146,69 @@ make resolve-local
 
 ### Generate code during local development
 
-> More details on scaffolding code using kubebuilder can be found [here](https://github.com/kubernetes-sigs/kubebuilder/blob/master/designs/simplified-scaffolding.md).
+Learn more about [Simplified Builder-Based Scaffolding](https://github.com/kubernetes-sigs/kubebuilder/blob/master/designs/simplified-scaffolding.md).
```

Comment:
Can we put the `kubebuilder` name back in? Kubebuilder is a tool that's well known, whereas simplified builder-based scaffolding does not ring a bell (at least for me)

Reply:
Sure! I thought it's omplied in "builder-based", but it's when that's not clear enough we absolutely need to add it.